src/main/java/org/cloudbees/literate/jenkins/LiterateEnvironmentBuild.java

```diff
@@ -202,7 +202,7 @@ protected Result doRun(final BuildListener listener) throws Exception, RunnerAbo
             ProjectModel model = projectModelAction.getModel();
             Map<Class<? extends Publisher>, List<Agent<? extends Publisher>>> agents = new LinkedHashMap<Class<?
                     extends Publisher>, List<Agent<? extends Publisher>>>();
-            for (Agent agent : Jenkins.getInstance().getExtensionList(Agent.class)) {
+            for (Agent agent : ExtensionList.lookup(Agent.class)) {
```

Comment:
Is there any difference between this two calls? (more than avoiding null pointer on ```Jenkins.getInstance()```)